### PR TITLE
Add helpful info to doco for use within symfony.

### PR DESCRIPTION
**Pull request by joshpme**
Some extra doco I would have found extremely helpful when using this repo.

### CODE REVIEW

File: doc/usage.md [hunk 1]
```diff
@@ -5,6 +5,22 @@ Before starting, if you're not already familiar with the Symfony2 form component
...
+    form.resolved_type_factory:
+        class: Ivory\OrderedForm\OrderedResolvedFormTypeFactory
+
+    ordered.extension:
+        class: Ivory\OrderedForm\Extension\OrderedExtension
+        tags:
+          - { name: form.type_extension }
+```
+
+
```
Comment (egeloen) [inline doc/usage.md]: Can you remove this extra blank line?

**Comment by egeloen** [comment]
Thanks! Just fix my comment and I merge it :)

**Comment by joshpme** [comment]
I've made a few more modifications.

**Comment by egeloen** [comment]
Just a quick question: Do you know that there is a bundle for this library? Basically, the bundle does what you are documenting :)

**Comment by joshpme** [comment]
No I didn't!? Is the bundle in your repo? how do I load it?

**Comment by joshpme** [comment]
Found it, thanks so much. Saves me alot of hassle :)

**Comment by egeloen** [comment]
Closing as such informations are not needed, just use the bundle :)

**Pull request closed without merging**